unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FeatureController/FeatureTemplate/Feature.cs

```diff
@@ -2,7 +2,18 @@
 using System.Collections.Generic;
 using UnityEngine;
 
+/// <summary>
+/// This is the main template for the features handled by FeatureController
+///
+/// This will handle all the monobehaviour callbacks when your feature is activated
+/// </summary>
 public class Feature
```

Comment:
If this class will not have any implementation, why don't we convert it into an interface? or into an abstract class al least?